config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java

```diff
@@ -355,7 +355,43 @@ public void shouldLoadForksIgnoreClassicForkBlock() throws IOException {
                         StandardCharsets.UTF_8)));
     final GenesisConfigFile config = fromConfig(configNode);
 
-    assertThat(config.getForks()).containsExactly(1L, 2L, 3L, 3L, 1035301L);
+    assertThat(config.getForks()).containsExactly(1L, 2L, 3L, 1035301L);
+    assertThat(config.getConfigOptions().getChainId()).hasValue(BigInteger.valueOf(61));
+  }
+
+  @Test
+  public void shouldLoadForksIgnoreUnexpectedValues() throws IOException {
+    final ObjectNode configNode =
+        new ObjectMapper()
+            .createObjectNode()
+            .set(
+                "config",
+                JsonUtil.objectNodeFromString(
+                    Resources.toString(
+                        Resources.getResource(
+                            // If you inspect this config, you should see that classicForkBlock is
+                            // declared (which we want to ignore)
+                            "valid_config_with_etc_forks.json"),
+                        StandardCharsets.UTF_8)));
+
+    final ObjectNode configNodeWithUnexpectedForks =
+        new ObjectMapper()
+            .createObjectNode()
+            .set(
+                "config",
+                JsonUtil.objectNodeFromString(
+                    Resources.toString(
+                        Resources.getResource(
+                            // If you inspect this config, you should see that classicForkBlock is
+                            // declared (which we want to ignore)
+                            "valid_config_with_unexpected_forks.json"),
+                        StandardCharsets.UTF_8)));
+
+    final GenesisConfigFile config = fromConfig(configNode);
+    final GenesisConfigFile configWithUnexpectedForks = fromConfig(configNodeWithUnexpectedForks);
+
+    assertThat(config.getForks()).containsExactly(1L, 2L, 3L, 1035301L);
```

Comment:
Need to add back in the assertion that the result is the same as that generated from the config with no unexpected fork values

Reply:
Are you suggesting the assertion on line 394?

Reply:
Yes it is there.